cmd/mtnormalise.cpp

```diff
@@ -64,16 +66,25 @@ void usage ()
     + Option ("niter", "set the number of iterations. (default: " + str(DEFAULT_MAIN_ITER_VALUE) + ")")
     + Argument ("number").type_integer()
 
+    + Option ("value", "specify the (positive) reference value to which the summed tissue compartments will be normalised. "
+                       "(default: " + str(DEFAULT_NORM_VALUE, 6) + ", SH DC term for unit angular integral)")
+    + Argument ("number").type_float (std::numeric_limits<default_type>::min())
+
+    + Option ("balanced", "incorporate the per-tissue balancing factors into scaling of the output images "
+                          "(NOTE: use of this option has critical consequences for AFD intensity normalisation; "
+                          "should not be used unless these consequences are fully understood)")
+
+    + OptionGroup ("Options for outputting data to verify successful operation of the mtnormalise command")
```

Comment:
That feels a big long. How about simply "debugging options"?

Reply:
Trouble is they may not be used solely for debugging :smiling_imp:

Reply:
I agree with @jdtournier here; I'm keen on seeing a better justification for this than just ":smiling_imp:".  People can be creative however they like, but it doesn't mean we have to endorse using certain outputs for certain purposes without a justification.  There's a certain context within which these numbers exactly live (Dave, Rami and myself *very* carefully thought about this and discussed this at length at the time; and for very good reasons), and using them outside that context can actually be quite misleading in this very case.  So well, happy to understand the justification before I can endorse this.